src/commands/build/wranglerjs/mod.rs

```diff
@@ -122,7 +122,9 @@ fn write_wranglerjs_output(
 ) -> Result<(), failure::Error> {
     if output.has_errors() {
         message::user_error(output.get_errors().as_str());
-        failure::bail!("Webpack returned an error");
+        failure::bail!(
+            "Webpack returned an error. Try re-running npm install, or yarn check --verify-tree."
```

Comment:
will running `yarn check --verify-tree` create a `yarn.lock` file? i'd hesitate to recommend using `yarn` when wrangler uses `npm`

Reply:
I think I'd just add `"You may be able to resolve this issue by running `npm install`"`

Reply:
yah wasn't sure bout the yarn thing, snagged it from the original issue discussion to see how people reacted. i think that's a good text.